context-free.md

```diff
@@ -1,22 +1,37 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+This programming language is for people who want to create digital art.
 
 ## What is easy to do in this language? Why is it easy?
-
+Recursively creating shapes is easy because shapes are created inside rules.
 
 ## What is hard to do in this language? Why is it hard?
-
+It was very hard for me to learn it, taking well over an hour.  It requires
+memorization of syntax and avaiable rules unlike a simple program such as paint
+where you can see what is available for use.  Additional memorization is
+required to know what transforamtions you have available for your shape (such
+as using hue to color things is less obvious than something like color). Drawing
```

Comment:
Great point here! I hadn't considered the fact that only offering the HSV space and forcing the user to perform the transformation to RGB values themselves makes the use of color perhaps a little less intuitive than need be.